presentation/src/main/java/com/teamagam/gimelgimel/presentation/presenters/SendGeoEntityPresenter.java

```diff
@@ -0,0 +1,121 @@
+package com.teamagam.gimelgimel.presentation.presenters;
+
+import com.teamagam.gimelgimel.domain.geometries.GeometryInteractor;
+import com.teamagam.gimelgimel.domain.geometries.entities.GeoEntity;
+import com.teamagam.gimelgimel.domain.geometries.entities.Geometry;
+import com.teamagam.gimelgimel.domain.geometries.entities.PointGeometry;
+import com.teamagam.gimelgimel.domain.geometries.entities.Symbol;
+import com.teamagam.gimelgimel.domain.messages.entity.MessageGeo;
+import com.teamagam.gimelgimel.presentation.presenters.base.AbstractPresenter;
+import com.teamagam.gimelgimel.presentation.presenters.base.BaseView;
+import com.teamagam.gimelgimel.presentation.rx.subscribers.BaseSubscriber;
+
+public class SendGeoEntityPresenter extends AbstractPresenter {
+
+    private GeometryInteractor mGeometryInteractor;
+    private View mView;
+
+    public SendGeoEntityPresenter(GeometryInteractor geometryInteractor, View view) {
+        mGeometryInteractor = geometryInteractor;
+        mView = view;
+    }
+
+    @Override
+    public void resume() {
+        mView.hideProgress();
+    }
+
+    @Override
+    public void pause() {
+        mView.hideProgress();
+    }
+
+    @Override
+    public void stop() {
+        mView.hideProgress();
+    }
+
+    @Override
+    public void destroy() {
+        mView.hideProgress();
+    }
+
+    public void sendGeoMessage(String senderId, String messageText, double latitude,
+                               double longitude, double altitude, String type) {
+
+        PointGeometry geometry = new PointGeometry(latitude, longitude, altitude);
+        GeoEntity geoEntity = createGeoEntity(senderId + messageText + type, geometry, type);
+        MessageGeo message = new MessageGeo(senderId, geoEntity, messageText, type);
+
+        mGeometryInteractor.addUserGeoEntity(message, new GeoSubscriber());
+    }
+
+    /**
+     * Temp implementation of GeoEntity
+     * @param id
+     * @param geometry
+     * @return
+     */
+    private GeoEntity createGeoEntity(String id, Geometry geometry, String type) {
```

Comment:
I dont think you should leave this temp here. it is part of the task to define the domain (including GeoEntity Basic implementation)